### ISSUE TITLE
Attempt to call method 'CovenantToFactionID' (a nil value)

### ISSUE THREAD

**Issue by Rychard** [Myrroddin/repbyzone#3]
Received the following error after updating:
```
..\AddOns\RepByZone\Core-BCC.lua line 119: attempt to call method 'CovenantToFactionID' (a nil value)
```

https://github.com/Myrroddin/repbyzone/blob/3546aa45817b6880399b16bba4868189ff7eb466/Core-BCC.lua#L119

The local being assigned doesn't appear to be referenced elsewhere, and commenting out the line seems to resolve the issue on my end.

**Comment by Myrroddin** [comment]
Will be fixed in 1.19.6, whenever GitHub finishes processing.

**Comment by Myrroddin** [comment]
Fixed with 1.19.6
